main.py

```diff
@@ -1,25 +1,75 @@
 from flask import Flask
-from models import Whisky, db
+from models import Whisky, db, Taste
 from flask import render_template, request, redirect
 
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///whisky.db'
 db.init_app(app)
 app.app_context().push()
 
-@app.route('/', methods=['POST', 'GET'])
+@app.route('/')
 def hello_world():
-    whiskies = Whisky.query.limit(3).all()
+    whiskies = Whisky.query.all()
     return render_template('index.html', whisky=whiskies)
 
 @app.route('/<int:id>', methods=['POST', 'GET'])
 def whisky(id):
     whisky = Whisky.query.get_or_404(id)
+    tastes = []
```

Comment:
tastes = [taste.name for taste i whisky.tastes]
Tak, żeby było bardziej pythonic 👍